topi/python/topi/cuda/softmax.py

```diff
@@ -52,13 +60,22 @@ def schedule_softmax(outs):
         raise ValueError('Tag is expected to be softmax_output or log_softmax_output. \
                          Got {0}'.format(op_tag))
 
+    # create tuning space
+    max_num_threads = tvm.target.current_target(allow_none=False).max_num_threads
+    possible_num_thread = get_powers_of_two_in_range(32, max_num_threads)
```

Comment:
A minor issue, but if max num threads is less than 32 this returns an empty list.
And max num threads is not necessary a power of two